### ISSUE TITLE
500 Error when user has no rooms but is allowed to create new room

### ISSUE THREAD

**Issue by stonerl** [bigbluebutton/greenlight#1749]
I'm not sure whether this is a greenlight or a bigbluebutton bug.

I limited the default `User` account to not be able to create new rooms, therefore when a user registers no `Starting Room` is created. I created a second role called `Moderator`. This role allows users to create new rooms. 

The problem is as follows. Since the user has no starting room, elevating him to a moderator results in a `500 server error`. The user is also not able to create new rooms although he has the privilege to do so.

To reproduce this:

1. Remove `Can create rooms` from the default `User` role.
2. Create a new role called `Moderator`
3. Add `Can create rooms` to this role.
4. A new user signs up and gets the default `User` role assigned.
5. Assign the `Moderator`role  to the new User.
6. Log the newly created user out and in again.
7. The user gets an `500` since he has no rooms.

Server: Ubuntu 16.04
BBB:: 2.2.16
Greenlight: 2.6.3
Tested with greenlight `2.6.3`& bigbluebutton

**Comment by lakano** [comment]
We have the same problem.
In log: 
`2020/06/04 12:25:48 [error] 6962#6962: *27523 open() "/var/www/bigbluebutton-default/owa/auth/logon.aspx" failed (2: No such file or directory), client: 162.243.142.x, server: gnfa-controletek.c2c.video, request: "GET /owa/auth/logon.aspx?url=https%3a%2f%2f1%2fecp%2f HTTP/1.1", host: "51.178.94.x"
`
And after an updatedb / locate 'logon.aspx', this file is nowhere on the disk.
In our case, this happen only on this new version of BBB on a fresh install of yesterday. Our old BBB servers (upgraded ~ 10 days but not to theses lasts versions doesn't have this problem).

**Comment by farhatahmad** [comment]
Yeah I was able to reproduce this locally. Going to tag it as a bug and get it fixed in the next release

**Comment by lakano** [comment]
@farhatahmad Please, do you have an idea of workaround until the next release, we have a lots of users who can't connect to receives participants and make the formations

**Comment by lakano** [comment]
Thanks a lot @farhatahmad for your reactivity and this new release !

**Comment by stonerl** [comment]
Big thanks from my side as well.

**Comment by lakano** [comment]
BTW, account created before this patch continue to have the error 500, is it normal or do I need to do something special ( docker-compose down && docker pull bigbluebutton/greenlight:v2 && docker-compose up -d ) ? The version on the bottom of the page is the last one ( 2.6.4 )
